src/sync-client.js

```diff
@@ -910,6 +911,11 @@ var self = {
 
   doCloudCall: function(params, success, failure) {
     if(self.cloudHandler && typeof self.cloudHandler === "function" ){
+      if (params && params.req && params.req.__fh) {
```

Comment:
Thanks for checking this! 
We should remove that. I also found that when testing it :D